src/assets/stylesheets/style.css

```diff
@@ -8676,7 +8697,7 @@ body {
 .nav-tabs > li {
   float: none;
   margin-bottom: -1px;
-  width: 100px;
```

Comment:
This might cause issues in other tabs. Auto should usually do the job.

Reply:
Auto was misbehaving initially, so have added 100px... Because of modifying a few more CSS, now auto seems to be working fine.